core/src/text/fontContext.cpp

```diff
@@ -296,20 +296,6 @@ void FontContext::releaseFonts() {
     std::lock_guard<std::mutex> lock(m_fontMutex);
     // Unload Freetype and Harfbuzz resources for all font faces
     m_alfons.unload();
-
-    // Release system font fallbacks input source data from default fonts, since
-    // those are 'weak' resources (would be automatically reloaded by alfons from
-    // its URI or source callback.
-    for (auto& font : m_font) {
-        if (!font) { continue; }
-        for (auto& face : font->faces()) {
-            alfons::InputSource& fontSource = face->descriptor().source;
```

Comment:
Comment says you removed `clearData` from `alfons::InputSource`, should there be a commit to update alfons submodule here?

Reply:
Updating alfons is not needed for this PR